typescript/phoenix-sdk/examples/placeLimitOrder.ts

```diff
@@ -90,11 +90,12 @@ export async function placeLimitOrderExample() {
     );
 
     // Create a limit order instruction
-    const limitOrderIx = PhoenixSdk.getLimitOrderIx(
-      marketAddress,
-      marketData,
-      traderKeypair.publicKey,
-      limitOrderPacket
+    const limitOrderIx = phoenixClient.createPlaceLimitOrderInstruction(
+      {
+        orderPacket: limitOrderPacket,
+      },
+      marketAddress.toBase58(),
```

Comment:
Is there a reason we prefer the wrapper object `PlaceLimitOrderInstructionArgs` syntax versus just passing in a `limitOrderPacket`? The latter feels simpler and not any less ambiguous.

Reply:
just changed this!